go.mod

```diff
@@ -3,29 +3,25 @@ module github.com/aquasecurity/kube-bench
 go 1.16
 
 require (
-	github.com/aws/aws-sdk-go v1.35.28
+	github.com/aws/aws-sdk-go v1.40.9
 	github.com/denisenkom/go-mssqldb v0.0.0-20190515213511-eb9f6a1743f3 // indirect
 	github.com/erikstmartin/go-testdb v0.0.0-20160219214506-8d10e4a1bae5 // indirect
-	github.com/fatih/color v1.5.0
+	github.com/fatih/color v1.12.0
 	github.com/go-sql-driver/mysql v1.4.1 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/imdario/mergo v0.3.5 // indirect
 	github.com/jinzhu/gorm v0.0.0-20160404144928-5174cc5c242a
 	github.com/jinzhu/inflection v0.0.0-20170102125226-1c35d901db3d // indirect
 	github.com/jinzhu/now v1.0.1 // indirect
 	github.com/lib/pq v0.0.0-20171126050459-83612a56d3dd // indirect
-	github.com/magiconair/properties v1.8.0
-	github.com/mattn/go-colorable v0.0.0-20170210172801-5411d3eea597 // indirect
-	github.com/mattn/go-isatty v0.0.0-20170307163044-57fdcb988a5c // indirect
+	github.com/magiconair/properties v1.8.5
 	github.com/mattn/go-sqlite3 v1.10.0 // indirect
-	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/ginkgo v1.16.4
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.4.0
-	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
-	google.golang.org/appengine v1.5.0 // indirect
-	gopkg.in/yaml.v2 v2.2.8
+	github.com/spf13/cobra v0.0.7
+	github.com/spf13/viper v1.8.1
+	github.com/stretchr/testify v1.7.0
+	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
```

Comment:
There is now a v0.21.3 for k8s.io/api , k8s.io/apimachinery and k8s.io/client-go but it requires a lot more adaptations

Reply:
I think then that we should wait for a major release, right now its working and I don't see the value for updating it if it will requires a lot more adaptations. 
Lets cross this bridge when they will release version 1